app/views/requirements/_form.html.haml

```diff
@@ -15,5 +15,9 @@
     = f.label(:description)
     = f.text_field(:description)
 
+  .fill-me-in
+    = f.label("Estimate in days")
```

Comment:
Use can use [active record translations](http://stackoverflow.com/questions/10812774/how-can-i-translate-the-activerecord-attribute-name-in-rails-3-2) here.